lib/authSession.ts

```diff
@@ -0,0 +1,17 @@
+import { auth as useAuthSession } from '@/auth'
+
+export const auth = async (requestOrContext) => {
```

Comment:
Is it possible to mock `requestOrContext` so that you can simplify this?

Reply:
Put down what you found in your investigation here

Reply:
As of now it is a argument I am using, I don't have the knowledge to figure this out yet.